x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/exception_item_card/meta.tsx

```diff
@@ -51,6 +51,11 @@ export const ExceptionItemCardMetaInfo = memo<ExceptionItemCardMetaInfoProps>(
     const onCloseRulesPopover = () => setIsRulesPopoverOpen(false);
     const onClosListsPopover = () => setIsListsPopoverOpen(false);
 
+    const isExpired = useMemo(
+      () => (item.expire_time ? new Date(item.expire_time) <= new Date() : false),
+      [item]
+    );
```

Comment:
can we import this function from your changes here?

`packages/kbn-securitysolution-exception-list-components/src/exception_item_card/meta/index.tsx`